hack/graph-util.py

```diff
@@ -472,18 +471,17 @@ def get_token(args):
     return None
 
 def set_log_level(args):
-    if args.verbose == "info":
-        _LOGGER.setLevel(logging.INFO)
-        return
-    _LOGGER.setLevel(logging.DEBUG)
+    _LOGGER.setLevel(logging.INFO)
+    if args.verbose == "debug":
+        _LOGGER.setLevel(logging.DEBUG)
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Utilities for managing graph data.')
     parser.add_argument(
         '-v', '--verbose',
```

Comment:
nit: `--log-level` extends more gracefully if we ever wanted to support `WARNING`, etc.  But since this script is going away soon (we hope), probably doesn't matter either way.